core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -150,6 +150,7 @@
   MASTER_WEB_HOSTNAME(Name.MASTER_WEB_HOSTNAME, null),
   MASTER_WEB_PORT(Name.MASTER_WEB_PORT, 19999),
   MASTER_WHITELIST(Name.MASTER_WHITELIST, "/"),
+  MASTER_CONNECTION_TIMEOUT_MS(Name.MASTER_CONNECTION_TIMEOUT_MS, 0),
```

Comment:
What does a value of 0 do for thrift?

Also, there should be documentation updates for this new parameter.